### PR TITLE
WAN Assurance plugin 3.7.1 release notes

### PR DESCRIPTION
**Pull request by agrawalkaushik**
DOCS-354 #time 20m

### CODE REVIEW

**Review by BenMatase — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by gregschrock — APPROVED** [review]

File: docs/release_notes_wan_assurance_plugin_3.7.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
 sidebar_label: '3.7'
 ---
+## Release 3.7.1
+**Release Date:** Mar 31, 2023
+
+### Resolved Issues
+
+- **WAN-1698 Classify custom apps on the hub routers based on the service profile**
+
+  _**Resolution:**_ Any service with RFC1918 address space will be tagged as custom app in the data that hub sends to MIST cloud.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_wan_assurance_plugin_3.7.md:12]: _**Resolution:**_ Any service with RFC1918 address space will be tagged as a custom app in the data that the hub sends to the MIST cloud.

File: docs/release_notes_wan_assurance_plugin_3.7.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+
+- **WAN-1737 De-duplicate peer path down events**
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_wan_assurance_plugin_3.7.md:14]: - **WAN-1737 Prevent duplication of peer path down events**

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

File: docs/release_notes_wan_assurance_plugin_3.7.md (hunk 1 above)
Comment (agrawalkaushik) [reply docs/release_notes_wan_assurance_plugin_3.7.md:12]: sorry having a hard time spotting the correction in your suggestion @Chr1st0ph3rTurn3r but I'll just take it as is 😄

Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_wan_assurance_plugin_3.7.md:12]: No worries - just missing articles: ...**a** custom app  ...that **the** hub sends to **the** Mist cloud.

**Pull request merged**